<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8"/>
<title></title>
<meta name="author" content="(Arvind Subramaniam)"/>
<style type="text/css">
.underline { text-decoration: underline; }
</style>
<link rel="stylesheet" href="./reveal.js/css/reveal.css"/>

<link rel="stylesheet" href="./reveal.js/css/theme/default.css" id="theme"/>


<!-- If the query includes 'print-pdf', include the PDF print sheet -->
<script>
    if( window.location.search.match( /print-pdf/gi ) ) {
        var link = document.createElement( 'link' );
        link.rel = 'stylesheet';
        link.type = 'text/css';
        link.href = './reveal.js/css/print/pdf.css';
        document.getElementsByTagName( 'head' )[0].appendChild( link );
    }
</script>
</head>
<body>
<div class="reveal">
<div class="slides">

<section  class="center">
<h1 id="org9798a13">Data Science Using R</h1>
<p>
Arvind R. Subramaniam
</p>

<p>
Assistant Member
</p>

<p>
Basic Sciences Division and Computational Biology Program
</p>

<p>
Fred Hutchinson Cancer Research Center
</p>

</section>
<section >
<h1 id="org53f304b">What you will learn over the next 3 lectures</h1>
<ul>
<li>Loading, Transforming, Visualizing Tabular Data (<a href="https://www.tidyverse.org/">Tidyverse</a>)
<ul>
<li><a href="https://readr.tidyverse.org/">readr</a></li>
<li><a href="http://dplyr.tidyverse.org">dplyr</a></li>
<li><a href="http://ggplot2.tidyverse.org">ggplot2</a></li>

</ul></li>
<li>Working with High-Throughput Genomic Data (<a href="http://bioconductor.org">Bioconductor</a>)
<ul>
<li><a href="https://bioconductor.org/packages/release/bioc/html/GenomicRanges.html">GenomicRanges</a></li>
<li><a href="https://bioconductor.org/packages/release/bioc/html/Biostrings.html">Biostrings</a></li>
<li><a href="https://bioconductor.org/packages/release/bioc/html/GenomicAlignments.html">GenomicAlignments</a></li>

</ul></li>

</ul>

<aside class="notes">
<ul>
<li>You will learn specific examples of how to do things, but also some general data analysis principles</li>

</ul>

</aside>
</section>
<section >
<h1 id="org4f9fa87">Example Datasets</h1>
<ul>
<li>Plate Reader Assay</li>
<li>Flow Cytometry</li>
<li>RNA-Seq</li>

</ul>

</section>
<section >
<h1 id="org6a7b424">Raw Flow Cytometry Data</h1>
<table id="org8b69bd0" border="0" cellspacing="0" cellpadding="6" rules="none" frame="none" width="100%" class="shrunk">


<colgroup>
<col  class="org-right" />

<col  class="org-right" />

<col  class="org-right" />

<col  class="org-right" />

<col  class="org-right" />
</colgroup>
<thead>
<tr>
<th scope="col" class="org-right">FSC.A</th>
<th scope="col" class="org-right">SSC.A</th>
<th scope="col" class="org-right">FITC.A</th>
<th scope="col" class="org-right">PE.Texas.Red.A</th>
<th scope="col" class="org-right">Time</th>
</tr>
</thead>
<tbody>
<tr>
<td class="org-right">79033</td>
<td class="org-right">69338</td>
<td class="org-right">9173</td>
<td class="org-right">18690</td>
<td class="org-right">3.02</td>
</tr>

<tr>
<td class="org-right">101336</td>
<td class="org-right">87574</td>
<td class="org-right">13184</td>
<td class="org-right">29886</td>
<td class="org-right">3.04</td>
</tr>

<tr>
<td class="org-right">51737</td>
<td class="org-right">56161</td>
<td class="org-right">3083</td>
<td class="org-right">18324</td>
<td class="org-right">3.06</td>
</tr>

<tr>
<td class="org-right">79904</td>
<td class="org-right">45085</td>
<td class="org-right">9957</td>
<td class="org-right">18099</td>
<td class="org-right">3.08</td>
</tr>

<tr>
<td class="org-right">124491</td>
<td class="org-right">97305</td>
<td class="org-right">15739</td>
<td class="org-right">28730</td>
<td class="org-right">3.09</td>
</tr>

<tr>
<td class="org-right">54359</td>
<td class="org-right">45015</td>
<td class="org-right">6175</td>
<td class="org-right">11918</td>
<td class="org-right">3.11</td>
</tr>

<tr>
<td class="org-right">64615</td>
<td class="org-right">88989</td>
<td class="org-right">11907</td>
<td class="org-right">32413</td>
<td class="org-right">3.13</td>
</tr>

<tr>
<td class="org-right">109592</td>
<td class="org-right">64132</td>
<td class="org-right">12561</td>
<td class="org-right">18824</td>
<td class="org-right">3.15</td>
</tr>

<tr>
<td class="org-right">58503</td>
<td class="org-right">116384</td>
<td class="org-right">11591</td>
<td class="org-right">27629</td>
<td class="org-right">3.19</td>
</tr>

<tr>
<td class="org-right">38634</td>
<td class="org-right">51511</td>
<td class="org-right">7200</td>
<td class="org-right">21930</td>
<td class="org-right">3.21</td>
</tr>
</tbody>
</table>

<p>
<code>5 cols × 2,720,000 rows</code>
</p>


</section>
<section >
<h1 id="org389c45a">Flow Cytometry Analysis Using Tidyverse</h1>

<div class="figure">
<p><img src="img/example_flow_cytometry_analysis.png" alt="example_flow_cytometry_analysis.png" width="900px" />
</p>
</div>


</section>
<section >
<h1 id="org7fa7d60">Raw Deep Sequencing Data</h1>
<p class="verse">
TACCTGGTTGATCCTGCCAGTAGCATATGCTTGTCTAAAAANAAAAAAAA<br />
TCGGGGCCCGAAGCGTTTACTTTGAAATAAAAAAAAAAAAAAAAAAAGAT<br />
ATTGATCATCGACACTTCGAACGCACTTGCGGCCCCGGGTAAAAAAAAAA<br />
TCATTGATCATCGACACTTCGAACGCACTTGCGGCCCCGGGTAAAAAAAA<br />
ATATTAAATGGATTTTTGGAGCAGGGAGATGGAAAAAAAAAAAAAAAAAA<br />
TCCCTGGTGGTCTAGTGGTTAGGATTCAAAAAAAAAAAAAAAAAAAGATC<br />
TCGTCGCGCGCGCGTCCGCTGGGGGCGGGGAGCGGTAAAAAAAAAAAAAA<br />
TCGCGCCGTCGGGCCAAAAAAAAAAAAAAAAAAAGATCGGAAGAGCACAC<br />
ATAAAGGTGCCCGGCGCCGGCTTCAAAAAAAAAAAAAAAAAAAAGATCGG<br />
TGGTGGCGGCGGCGGCGGCGGCGGCGGGAAAAAAAAAAAAAAAAAAAGAT<br />
TTGACTGCATAATTTGTGGTAGTGAGGGAAAAAAAAAAAAAAAAAAAAAT<br />
TAGAGTGAGACTCCGTCTGCAAAAAAAAAAAAAAAAAAAAAGATCGGAAG<br />
CCGAGGGAGCGAGACCCAAAAAAAAAAAAAAAAAAAAAAATCGGAAAAGC<br />
ATTTTCAAGAAGATCATCTTTTAGACCAGCACTGGAGAAAAAAAAAAAAA<br />
ATTGATCATCGACACTTCGAACGCACTTGCGGCCCCGGGAAAAAAAAAAA<br />
</p>

<p>
<code>165,000,000 reads</code>
</p>


</section>
<section >
<h1 id="org1148dda">Deep Sequencing Analysis Using Bioconductor</h1>

<div class="figure">
<p><img src="img/example_riboseq_analysis.png" alt="example_riboseq_analysis.png" width="1200px" />
</p>
</div>


</section>
<section >
<h1 id="org332882d">Deep Sequencing Analysis Using Bioconductor</h1>

<div class="figure">
<p><img src="img/example_riboseq_analysis_2.png" alt="example_riboseq_analysis_2.png" width="600px" />
</p>
</div>


</section>
<section >
<h1 id="orgdf02333"><code>Tidyverse</code> Functions for Tabular Data</h1>
<table border="0" cellspacing="0" cellpadding="6" rules="none" frame="none" style="margin-top:100px;width:100%;">


<colgroup>
<col  class="org-left" />

<col  class="org-left" />

<col  class="org-left" />
</colgroup>
<thead>
<tr>
<th scope="col" class="org-left">Import</th>
<th scope="col" class="org-left">Visualize</th>
<th scope="col" class="org-left">Transform</th>
</tr>
</thead>
<tbody>
<tr>
<td class="org-left"><code>read_tsv</code></td>
<td class="org-left"><code>geom_point</code></td>
<td class="org-left"><code>select</code></td>
</tr>

<tr>
<td class="org-left">&#xa0;</td>
<td class="org-left"><code>geom_line</code></td>
<td class="org-left"><code>filter</code></td>
</tr>

<tr>
<td class="org-left">&#xa0;</td>
<td class="org-left"><code>facet_grid</code></td>
<td class="org-left"><code>arrange</code></td>
</tr>
</tbody>
</table>


</section>
<section >
<h1 id="orge36332d">☞ <b>Use <code>tsv</code> and <code>csv</code>  file formats for tabular data</b></h1>
<p>
<b>T</b>ab <b>S</b>eparated <b>V</b>alues
</p>


<p class="verse">
<br />
strain	mean_yfp mean_rfp mean_ratio se_ratio<br />
schp674	1270	20316	0.561	0.004	10×AAG	CAAA<br />
schp675	3687	20438	1.621	0.036	10×AAG	CCGC<br />
schp676	2657	20223	1.177	0.048	10×AAG	CCAA<br />
schp677	3967	20604	1.728	0.03	10×AAG	CCAC<br />
<br />
</p>

<p>
<b>C</b>omma <b>S</b>eparated <b>V</b>alues
</p>

<p class="verse">
<br />
strain,mean_yfp,mean_rfp,mean_ratio,se_ratio,insert_sequence,kozak_region<br />
schp674,1270,20316,0.561,0.004,10×AAG,CAAA<br />
schp675,3687,20438,1.621,0.036,10×AAG,CCGC<br />
schp676,2657,20223,1.177,0.048,10×AAG,CCAA<br />
schp677,3967,20604,1.728,0.03,10×AAG,CCAC<br />
<br />
</p>

<aside class="notes">
<ul>
<li>TSV and CSV formats can be displayed natively by github. Open this <a href="https://github.com/rasilab/mkriner_2018/tree/master/data/plate_reader">TSV</a> file on github and show how it is displayed nicely.</li>
<li>I personally prefer TSV since it is easier to align columns by eye.</li>
<li>These formats can be read by most programs in all operating systems.</li>
<li><code>.gz</code> and <code>.zip</code> compression can significantly reduce size.</li>

</ul>

</aside>


</section>
<section >
<h1 id="orgf289c07">Reading tabular data into R</h1>
<div class="org-src-container">

<pre  class="src src-R"><span style="color: #268bd2; font-weight: bold;">library</span>(tidyverse)

data <span style="color: #268bd2; font-weight: bold;">&lt;-</span> read_tsv(<span style="color: #2aa198;">"tables/example_dataset_1.tsv"</span>)
</pre>
</div>

<aside class="notes">
<ul>
<li>Introduce concept of assignment, function, and library</li>

</ul>

</aside>


</section>
<section >
<h1 id="org8a170a9">Read tabular data into a <code>DataFrame</code> (<code>tibble</code>)</h1>
<div class="org-src-container">

<pre  class="src src-R" id="flow_data"><span style="color: #268bd2; font-weight: bold;">library</span>(tidyverse)

data <span style="color: #268bd2; font-weight: bold;">&lt;-</span> read_tsv(<span style="color: #2aa198;">"tables/example_dataset_1.tsv"</span>)

print(data, n = 5)

</pre>
</div>

<pre class="example">
# A tibble: 16 x 7
  strain  mean_yfp mean_rfp mean_ratio se_ratio insert_sequence kozak_region
  &lt;chr&gt;      &lt;int&gt;    &lt;int&gt;      &lt;dbl&gt;    &lt;dbl&gt; &lt;chr&gt;           &lt;chr&gt;       
1 schp688     1748    20754      0.755    0.066 10×AGA          A           
2 schp684     3294    20585      1.44     0.021 10×AGA          B           
3 schp690     3535    20593      1.54     0.018 10×AGA          C           
4 schp687     4658    20860      2.00     0.021 10×AGA          D           
5 schp686     5000    21171      2.12     0.023 10×AGA          E           
# ... with 11 more rows

</pre>

<aside class="notes">
<ul>
<li><a href="example_dataset_1.tsv">example_dataset_1.tsv</a> created in <a href="file:///fh/fast/subramaniam_a/user/rasi/git/ribosome_colliisions_yeast/scripts/initiation_pgk1_10xaag_flow.Rmd">file:///fh/fast/subramaniam_a/user/rasi/git/ribosome_colliisions_yeast/scripts/initiation_pgk1_10xaag_flow.Rmd</a></li>

</ul>

</aside>

</section>
<section >
<h1 id="org49b5942">☞ <b>Comment your code</b></h1>
<div class="org-src-container">

<pre  class="src src-R"><span style="color: #586e75;"># </span><span style="color: #586e75;">library to work with tabular data</span>
<span style="color: #268bd2; font-weight: bold;">library</span>(tidyverse)

<span style="color: #586e75;"># </span><span style="color: #586e75;">read the tsv file into a tibble and </span>
<span style="color: #586e75;"># </span><span style="color: #586e75;">assign it to the 'data' variable</span>
data <span style="color: #268bd2; font-weight: bold;">&lt;-</span> read_tsv(<span style="color: #2aa198;">"tables/example_dataset_1.tsv"</span>)

<span style="color: #586e75;"># </span><span style="color: #586e75;">display the contents of 'data' </span>
print(data, n = 5)
</pre>
</div>


</section>
<section >
<h1 id="orgbbf4b63">Plotting a point graph</h1>
<div class="org-src-container">

<pre  class="src src-R">ggplot(data, aes(x = kozak_region,
                 y = mean_ratio)) +
  geom_point()

</pre>
</div>


<div class="figure">
<p><img src="img/ggplot2_point_example_no_color.png" alt="ggplot2_point_example_no_color.png" />
</p>
</div>

</section>
<section >
<h1 id="org23461e9">How do we show multiple experimental parameters?</h1>
<table border="0" cellspacing="0" cellpadding="6" rules="none" frame="none" width="100%" class="shrunk">


<colgroup>
<col  class="org-left" />

<col  class="org-right" />

<col  class="org-left" />

<col  class="org-left" />
</colgroup>
<thead>
<tr>
<th scope="col" class="org-left">strain</th>
<th scope="col" class="org-right">mean_ratio</th>
<th scope="col" class="org-left">insert_sequence</th>
<th scope="col" class="org-left">kozak_region</th>
</tr>
</thead>
<tbody>
<tr>
<td class="org-left">schp688</td>
<td class="org-right">0.755</td>
<td class="org-left">10×AGA</td>
<td class="org-left">A</td>
</tr>

<tr>
<td class="org-left">schp684</td>
<td class="org-right">1.437</td>
<td class="org-left">10×AGA</td>
<td class="org-left">B</td>
</tr>

<tr>
<td class="org-left">schp690</td>
<td class="org-right">1.541</td>
<td class="org-left">10×AGA</td>
<td class="org-left">C</td>
</tr>

<tr>
<td class="org-left">schp687</td>
<td class="org-right">2.004</td>
<td class="org-left">10×AGA</td>
<td class="org-left">D</td>
</tr>

<tr>
<td class="org-left">schp686</td>
<td class="org-right">2.121</td>
<td class="org-left">10×AGA</td>
<td class="org-left">E</td>
</tr>

<tr>
<td class="org-left">schp685</td>
<td class="org-right">2.893</td>
<td class="org-left">10×AGA</td>
<td class="org-left">F</td>
</tr>

<tr>
<td class="org-left">schp683</td>
<td class="org-right">3.522</td>
<td class="org-left">10×AGA</td>
<td class="org-left">G</td>
</tr>

<tr>
<td class="org-left">schp689</td>
<td class="org-right">3.424</td>
<td class="org-left">10×AGA</td>
<td class="org-left">H</td>
</tr>

<tr>
<td class="org-left">schp679</td>
<td class="org-right">1.149</td>
<td class="org-left">10×AAG</td>
<td class="org-left">A</td>
</tr>

<tr>
<td class="org-left">schp675</td>
<td class="org-right">1.621</td>
<td class="org-left">10×AAG</td>
<td class="org-left">B</td>
</tr>

<tr>
<td class="org-left">schp681</td>
<td class="org-right">1.645</td>
<td class="org-left">10×AAG</td>
<td class="org-left">C</td>
</tr>

<tr>
<td class="org-left">schp678</td>
<td class="org-right">1.906</td>
<td class="org-left">10×AAG</td>
<td class="org-left">D</td>
</tr>

<tr>
<td class="org-left">schp677</td>
<td class="org-right">1.728</td>
<td class="org-left">10×AAG</td>
<td class="org-left">E</td>
</tr>

<tr>
<td class="org-left">schp676</td>
<td class="org-right">1.177</td>
<td class="org-left">10×AAG</td>
<td class="org-left">F</td>
</tr>

<tr>
<td class="org-left">schp674</td>
<td class="org-right">0.561</td>
<td class="org-left">10×AAG</td>
<td class="org-left">G</td>
</tr>

<tr>
<td class="org-left">schp680</td>
<td class="org-right">0.519</td>
<td class="org-left">10×AAG</td>
<td class="org-left">H</td>
</tr>
</tbody>
</table>

</section>
<section >
<h1 id="orgda4b424">Plotting a point graph with color</h1>
<div class="org-src-container">

<pre  class="src src-R">ggplot(data, aes(x = kozak_region,
                 y = mean_ratio,
                 color = insert_sequence)) +
  geom_point()

</pre>
</div>


<div class="figure">
<p><img src="img/ggplot2_point_example.png" alt="ggplot2_point_example.png" />
</p>
</div>

</section>
<section >
<h1 id="orgfab67d7">Plotting a line graph</h1>
<div class="org-src-container">

<pre  class="src src-R">ggplot(data, aes(x = kozak_region,
                 y = mean_ratio,
                 color = insert_sequence,
                 group = insert_sequence)) +
  geom_line()

</pre>
</div>


<div class="figure">
<p><img src="img/ggplot2_line_example.png" alt="ggplot2_line_example.png" />
</p>
</div>

</section>
<section >
<h1 id="orgc7775e2">Plotting point and line graphs</h1>
<div class="org-src-container">

<pre  class="src src-R">ggplot(data, aes(x = kozak_region,
                 y = mean_ratio,
                 color = insert_sequence,
                 group = insert_sequence)) +
  geom_line() +
  geom_point()

</pre>
</div>


<div class="figure">
<p><img src="img/ggplot2_line_point_example.png" alt="ggplot2_line_point_example.png" />
</p>
</div>

<aside class="notes">
<ul>
<li>We see  that the two insert sequences have different trends as a function of initiation rate. How do we make this trend obvious? Connect them with a line!</li>

</ul>

</aside>
</section>
<section >
<h1 id="orge2604aa">‘Faceting’ &#x2013; Plotting in multiple panels</h1>
<div class="org-src-container">

<pre  class="src src-R">ggplot(data, aes(x = kozak_region,
                 y = mean_ratio,
                 group = insert_sequence)) +
  geom_line() +
  geom_point() +
  facet_grid(~ insert_sequence)
</pre>
</div>


<div class="figure">
<p><img src="img/ggplot2_line_point_facet_example.png" alt="ggplot2_line_point_facet_example.png" />
</p>
</div>

</section>
<section >
<h1 id="orgc71c376">☞ <b>Use the pipe <code>%&gt;%</code> operator to chain commands</b></h1>
<div class="org-src-container">

<pre  class="src src-R">print(data, n = 3)
</pre>
</div>

<pre class="example">
# A tibble: 16 x 7
  strain  mean_yfp mean_rfp mean_ratio se_ratio insert_sequence kozak_region
  &lt;chr&gt;      &lt;int&gt;    &lt;int&gt;      &lt;dbl&gt;    &lt;dbl&gt; &lt;chr&gt;           &lt;chr&gt;       
1 schp688     1748    20754      0.755    0.066 10×AGA          A           
2 schp684     3294    20585      1.44     0.021 10×AGA          B           
3 schp690     3535    20593      1.54     0.018 10×AGA          C           
# ... with 13 more rows

</pre>

<div class="org-src-container">

<pre  class="src src-R">data <span style="color: #268bd2; font-weight: bold;">%&gt;%</span>
  print(n = 3)
</pre>
</div>

<pre class="example">
# A tibble: 16 x 7
  strain  mean_yfp mean_rfp mean_ratio se_ratio insert_sequence kozak_region
  &lt;chr&gt;      &lt;int&gt;    &lt;int&gt;      &lt;dbl&gt;    &lt;dbl&gt; &lt;chr&gt;           &lt;chr&gt;       
1 schp688     1748    20754      0.755    0.066 10×AGA          A           
2 schp684     3294    20585      1.44     0.021 10×AGA          B           
3 schp690     3535    20593      1.54     0.018 10×AGA          C           
# ... with 13 more rows

</pre>

<aside class="notes">
<p>
Results from the LHS of the pipe are passed to the RHS.
</p>

</aside>

</section>
<section >
<h1 id="org1bef9d4">Simple Data Manipulations &#x2013; <code>select</code> columns</h1>
<div class="org-src-container">

<pre  class="src src-R">data <span style="color: #268bd2; font-weight: bold;">%&gt;%</span>
  print(n = 2)
</pre>
</div>

<pre class="example">
# A tibble: 16 x 7
  strain  mean_yfp mean_rfp mean_ratio se_ratio insert_sequence kozak_region
  &lt;chr&gt;      &lt;int&gt;    &lt;int&gt;      &lt;dbl&gt;    &lt;dbl&gt; &lt;chr&gt;           &lt;chr&gt;       
1 schp688     1748    20754      0.755    0.066 10×AGA          A           
2 schp684     3294    20585      1.44     0.021 10×AGA          B           
# ... with 14 more rows

</pre>

<div class="org-src-container">

<pre  class="src src-R">data <span style="color: #268bd2; font-weight: bold;">%&gt;%</span>
  select(strain, mean_ratio, insert_sequence, kozak_region) <span style="color: #268bd2; font-weight: bold;">%&gt;%</span>
  print(n = 2)
</pre>
</div>

<pre class="example">
# A tibble: 16 x 4
  strain  mean_ratio insert_sequence kozak_region
  &lt;chr&gt;        &lt;dbl&gt; &lt;chr&gt;           &lt;chr&gt;       
1 schp688      0.755 10×AGA          A           
2 schp684      1.44  10×AGA          B           
# ... with 14 more rows

</pre>


</section>
<section >
<h1 id="orga7d8baf">Simple Data Manipulations &#x2013; <code>filter</code>  rows</h1>
<div class="org-src-container">

<pre  class="src src-R">data <span style="color: #268bd2; font-weight: bold;">%&gt;%</span>
  filter(kozak_region == <span style="color: #2aa198;">"A"</span>)
</pre>
</div>

<pre class="example">
# A tibble: 2 x 7
  strain  mean_yfp mean_rfp mean_ratio se_ratio insert_sequence kozak_region
  &lt;chr&gt;      &lt;int&gt;    &lt;int&gt;      &lt;dbl&gt;    &lt;dbl&gt; &lt;chr&gt;           &lt;chr&gt;       
1 schp688     1748    20754      0.755    0.066 10×AGA          A           
2 schp679     2528    19906      1.15     0.056 10×AAG          A

</pre>

<div class="org-src-container">

<pre  class="src src-R">data <span style="color: #268bd2; font-weight: bold;">%&gt;%</span>
  filter(kozak_region == <span style="color: #2aa198;">"A"</span>, insert_sequence == <span style="color: #2aa198;">"10&#215;AGA"</span>)
</pre>
</div>

<pre class="example">
# A tibble: 1 x 7
  strain  mean_yfp mean_rfp mean_ratio se_ratio insert_sequence kozak_region
  &lt;chr&gt;      &lt;int&gt;    &lt;int&gt;      &lt;dbl&gt;    &lt;dbl&gt; &lt;chr&gt;           &lt;chr&gt;       
1 schp688     1748    20754      0.755    0.066 10×AGA          A

</pre>

<div class="org-src-container">

<pre  class="src src-R">data <span style="color: #268bd2; font-weight: bold;">%&gt;%</span>
  filter(kozak_region == <span style="color: #2aa198;">"A"</span>) <span style="color: #268bd2; font-weight: bold;">%&gt;%</span> 
  filter(insert_sequence == <span style="color: #2aa198;">"10&#215;AGA"</span>)
</pre>
</div>

<pre class="example">
# A tibble: 1 x 7
  strain  mean_yfp mean_rfp mean_ratio se_ratio insert_sequence kozak_region
  &lt;chr&gt;      &lt;int&gt;    &lt;int&gt;      &lt;dbl&gt;    &lt;dbl&gt; &lt;chr&gt;           &lt;chr&gt;       
1 schp688     1748    20754      0.755    0.066 10×AGA          A

</pre>

<aside class="notes">
<ul>
<li>Note composability</li>
<li>Note == sign</li>

</ul>

</aside>

</section>
<section >
<h1 id="org4de0323">Simple Data Manipulations &#x2013; <code>arrange</code>  rows</h1>
<div class="org-src-container">

<pre  class="src src-R">data <span style="color: #268bd2; font-weight: bold;">%&gt;%</span>
  arrange(mean_ratio)
</pre>
</div>

<pre class="example">
# A tibble: 16 x 7
   strain  mean_yfp mean_rfp mean_ratio se_ratio insert_sequence kozak_region
   &lt;chr&gt;      &lt;int&gt;    &lt;int&gt;      &lt;dbl&gt;    &lt;dbl&gt; &lt;chr&gt;           &lt;chr&gt;       
 1 schp680     1117    19377      0.519    0.01  10×AAG          H           
 2 schp674     1270    20316      0.561    0.004 10×AAG          G           
 3 schp688     1748    20754      0.755    0.066 10×AGA          A           
 4 schp679     2528    19906      1.15     0.056 10×AAG          A           
 5 schp676     2657    20223      1.18     0.048 10×AAG          F           
 6 schp684     3294    20585      1.44     0.021 10×AGA          B           
 7 schp690     3535    20593      1.54     0.018 10×AGA          C           
 8 schp675     3687    20438      1.62     0.036 10×AAG          B           
 9 schp681     3705    20227      1.64     0.021 10×AAG          C           
10 schp677     3967    20604      1.73     0.03  10×AAG          E           
11 schp678     4378    20630      1.91     0.01  10×AAG          D           
12 schp687     4658    20860      2.00     0.021 10×AGA          D           
13 schp686     5000    21171      2.12     0.023 10×AGA          E           
14 schp685     7379    22956      2.89     0.05  10×AGA          F           
15 schp689     8693    22649      3.42     0.125 10×AGA          H           
16 schp683     9365    23866      3.52     0.11  10×AGA          G
</pre>

<aside class="notes">
nil
</aside>

</section>
<section >
<h1 id="orgdcde14c">What you learned today</h1>
<table border="0" cellspacing="0" cellpadding="6" rules="none" frame="none" style="margin-top:100px;width:100%;">


<colgroup>
<col  class="org-left" />

<col  class="org-left" />

<col  class="org-left" />
</colgroup>
<thead>
<tr>
<th scope="col" class="org-left">Import</th>
<th scope="col" class="org-left">Visualize</th>
<th scope="col" class="org-left">Transform</th>
</tr>
</thead>
<tbody>
<tr>
<td class="org-left"><code>read_tsv</code></td>
<td class="org-left"><code>geom_point</code></td>
<td class="org-left"><code>select</code></td>
</tr>

<tr>
<td class="org-left">&#xa0;</td>
<td class="org-left"><code>geom_line</code></td>
<td class="org-left"><code>filter</code></td>
</tr>

<tr>
<td class="org-left">&#xa0;</td>
<td class="org-left"><code>facet_grid</code></td>
<td class="org-left"><code>arrange</code></td>
</tr>
</tbody>
</table>
</section>
</section>
</div>
</div>
<script src="./reveal.js/lib/js/head.min.js"></script>
<script src="./reveal.js/js/reveal.js"></script>

<script>
// Full list of configuration options available here:
// https://github.com/hakimel/reveal.js#configuration
Reveal.initialize({

controls: false,
progress: true,
history: true,
center: false,
slideNumber: 'c',
rollingLinks: false,
keyboard: true,
overview: true,

theme: Reveal.getQueryHash().theme, // available themes are in /css/theme
transition: Reveal.getQueryHash().transition || 'fade', // default/cube/page/concave/zoom/linear/fade/none
transitionSpeed: 'default',
multiplex: {
    secret: '', // null if client
    id: '', // id, obtained from socket.io server
    url: '' // Location of socket.io server
},

// Optional libraries used to extend on reveal.js
dependencies: [
 { src: './reveal.js/lib/js/classList.js', condition: function() { return !document.body.classList; } },
 { src: './reveal.js/plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
 { src: './reveal.js/plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
 { src: './reveal.js/plugin/zoom-js/zoom.js', async: true, condition: function() { return !!document.body.classList; } },
 { src: './reveal.js/plugin/notes/notes.js', async: true, condition: function() { return !!document.body.classList; } }]
});
</script>
</body>
</html>
