---
title: Arvind Rasi Subramaniam
mainfont: Helvetica
fontsize: 11
date:
linestretch: 1.2
---

\begin{center}
Associate Professor \\
Fred Hutchinson Cancer Center \\
\href{mailto:rasi@fredhutch.org}{rasi@fredhutch.org} \\ 
\url{https://rasilab.github.io}
\end{center}

\vskip 0.5cm 

# Education

\begin{tabular}{>{\raggedright\arraybackslash}p{5.8in} >{\raggedleft\arraybackslash}p{1.4in}} 
Ph.D. in Physics, University of Chicago &                                                           2008 \\
B.Tech. in Metallurgical and Materials Engineering, Indian Institute of Technology Madras, India  & 2004 \\
\end{tabular}

# Employment

\begin{tabular}{>{\raggedright\arraybackslash}p{5.8in} >{\raggedleft\arraybackslash}p{1.4in}} 
Associate Professor & 2021--\\
Assistant Professor \newline 
Basic Sciences Division \& Computational Biology Section of Public Health Sciences Division \newline
Fred Hutchinson Cancer Center, Seattle, WA \newline
&  2015--21 \\

Postdoctoral Fellow \newline
Center for Systems Biology \& Department of Molecular and Cellular Biology \newline
Harvard University, Cambridge, MA. Advisors: Erin O'Shea, Phillippe Cluzel \newline 
&   2008--15 \\
Graduate Fellow \newline
Theoretical Condensed Matter Physics Group \newline
University of Chicago, Chicago, IL. Advisor: Ilya Gruzberg 
& 2005--08 \\
\end{tabular}

# Other Appointments

\begin{tabular}{>{\raggedright\arraybackslash}p{6.6in} >{\raggedleft\arraybackslash}p{0.6in}} 
Affiliate Associate Professor, Department of Genome Sciences, University of Washington, Seattle & 2016--\\
Affiliate Associate Professor, Department of Biochemistry, University of Washington, Seattle & 2016--\\
Affiliate Associate Professor, Department of Microbiology, University of Washington, Seattle & 2018--\\
Participant in the Program on ‘Random Shapes’, Institute for Pure and Applied Mathematics, UCLA & 2007 \\
Visiting Affiliate in the Program on ‘Stochastic Geometry and Field Theory’, Kavli Institute for Theoretical Physics, UCSB & 2006 \\
Graduate Research Fellow, Experimental Condensed Matter Physics, University of Chicago, Advisor: Thomas Rosenbaum & 2004--05 \\
Summer Undergraduate Research Fellow, Material Science \& Engineering, California Institute of Technology, Advisor: Sossina Haile & 2003 \\
Summer Research Fellow, Department of Physics, Indian Institute of Science, Bangalore, Advisor: Arup Kumar Raychaudhuri & 2002 \\
\end{tabular}

# Honors

\begin{tabular}{>{\raggedright\arraybackslash}p{6.2in} >{\raggedleft\arraybackslash}p{1in}} 
Transformative Research Award, National Institutes of Health & 2023 \\
CAREER Award, National Science Foundation & 2018 \\
Sidney Kimmel Scholar & 2017 \\
K99/R00 Pathway to Independence Award, National Institutes of Health & 2013 \\
Wentzel Research Prize for Outstanding Research in Theoretical Physics, University of Chicago & 2007 \\
Chandrasekhar, McCormick, and Sachs Graduate Research Fellowships, University of Chicago & 2004 \\
Dhandapani Memorial Prize for highest GPA in Metallurgical and Materials Engineering, IIT Madras & 2004 \\
\end{tabular}

# Invited Talks

\begin{tabular}{>{\raggedright\arraybackslash}p{6.6in} >{\raggedleft\arraybackslash}p{0.6in}} 
EMBO Conference on RNA Meets Protein Decay, Vienna & 2025 \\
Departmental Seminar, Computational \& Systems Biology, University of Pittsburgh, Pittsburgh & 2025 \\
Departmental Seminar, Biochemistry and Molecular Biology, University of Chicago, Chicago & 2024 \\
Departmental Seminar, Chemistry and Biochemistry, University of California, San Diego & 2023 \\
Gordon Research Conference on Translational Machinery in Health and Disease, Galveston & 2023 \\
Emergent Simplicity in Biophysical Dynamics TSRC workshop, Telluride & 2019 \\
Department of Molecular Biology and Genetics, Johns Hopkins University School of Medicine, Baltimore & 2014 \\
\end{tabular}

# Teaching

\begin{tabular}{>{\raggedright\arraybackslash}p{6.6in} >{\raggedleft\arraybackslash}p{0.6in}} 
Lead Instructor, \href{https://github.com/FredHutch/tfcb_2024}{MCB 536: Tools for Computational Biology}, University of Washington, Seattle & 2018–25 \\
Teaching Assistant, Foundations of Systems Biology and Bioengineering, Harvard University & 2012 \\
Teaching Assistant, Introduction to Quantitative Tools for Cell Biology, Harvard University & 2010 \\
Teaching Assistant, Symplectic Methods of Classical Dynamics, University of Chicago & 2008 \\
Teaching Assistant, Advanced Mathematical Methods of Physics, University of Chicago & 2008 \\
Teaching Assistant, Solid State Physics, University of Chicago & 2007 \\
\end{tabular}

# Ph.D. Trainees

\begin{tabular}{>{\raggedright\arraybackslash}p{6.6in} >{\raggedleft\arraybackslash}p{0.6in}} 
Heather Borror, Molecular and Cellular Biology, University of Washington & 2024-- \\
Katharine Chen, Molecular and Cellular Biology, University of Washington & 2019--23 \\
Patrick Nugent, Molecular and Cellular Biology, University of Washington & 2018--25 \\
Ty Bottorff, Biophysics, Structure and Design Graduate Program, University of Washington & 2019--23 \\
Philip Burke, Microbiology, University of Washington (joint with Jesse Bloom) & 2017--22 \\
Heather Machkovech, Genome Sciences, University of Washington (joint with Jesse Bloom) & 2016--18 \\
\end{tabular}

# Postdoctoral Trainees

\begin{tabular}{>{\raggedright\arraybackslash}p{6.6in} >{\raggedleft\arraybackslash}p{0.6in}} 
Jamie Yelland & 2023-- \\
Matthew Chan (joint with Melody Campbell) & 2022-- \\
Maria Toro Moreno (joint with Harmit Malik) & 2021-- \\
Rachael Bakker & 2021--25 \\
Heungwon Park & 2016--19 \\
Michelle Kriner & 2016--18 \\
\end{tabular}

# Research Technician Trainees

\begin{tabular}{>{\raggedright\arraybackslash}p{6.6in} >{\raggedleft\arraybackslash}p{0.6in}} 
Shannon Marschall (joint with Akhila Rajan) & 2023--25 \\
Yuya Zhao & 2018--20 \\
Shea Ransford & 2017--18 \\
Michael ‘Max’ Ferrin & 2015--17 \\
\end{tabular}

# Ph.D. Thesis Committee 

\begin{longtable}{>{\raggedright\arraybackslash}p{6.3in} >{\raggedleft\arraybackslash}p{0.9in}} 
Serwah Danquah, Molecular Medicine and Mechanisms of Disease Program, University of Washington & 2025--\\
Kristian Davidsen, Molecular and Cellular Biology, University of Washington & 2020--24\\
Alison Greenlaw, Molecular and Cellular Biology, University of Washington & 2020--23\\
Alexandre Germanos, Molecular and Cellular Biology, University of Washington & 2019--23 \\
Samantha Schuster, Molecular and Cellular Biology, University of Washington & 2019--23 \\
Bianca Ruiz, Genome Sciences, University of Washington & 2018--21 \\
Dylan Udy, Molecular and Cellular Biology, University of Washington & 2017--22 \\
Joey Pangallo, Molecular and Cellular Biology, University of Washington & 2016--21 \\
Ethan Keeler, Electrical Engineering, University of Washington & 2016--18 \\
Robin Green, Molecular and Cellular Biology, University of Washington & 2015--17 \\
Qing Feng, Molecular and Cellular Biology, University of Washington & 2015--17 \\
\end{longtable}

# Internal Service

\begin{tabular}{>{\raggedright\arraybackslash}p{6.2in} >{\raggedleft\arraybackslash}p{1in}} 
Eddie Méndez Award Committee, Fred Hutchinson Cancer Research Center & 2021 \\
Faculty Search Committee, Basic Sciences Division, Fred Hutchinson Cancer Research Center & 2020,23 \\
Organizer, Computational Biology Seminar Series, Fred Hutchinson Cancer Research Center & 2017--21 \\
Weintraub Graduate Student Award Selection Committee, Fred Hutchinson Cancer Research Center & 2017,20 \\
Admissions Committee, Biophysics, Structure, \& Design Graduate Program, University of Washington & 2017--18 \\
Admissions Committee, Molecular and Cellular Biology Graduate Program, University of Washington & 2016,17,21 \\
Organizer, Microbial Sciences Initiative Journal Club, Harvard University & 2010 \\
Organizer, Metallurgical and Materials Engineering Student Association, IIT Madras, India & 2002--03 \\
\end{tabular}

# External Service

\begin{tabular}{>{\raggedright\arraybackslash}p{6.2in} >{\raggedleft\arraybackslash}p{1in}} 
Session chair, RNA Meets Protein Decay conference, Vienna & 2025\\
Ad-hoc member of of NIH ZRG1 MGG-F (55) panel for R35 MIRA & 2025\\
Member of NSF MCB panel & 2020,22 \\
Reviewer for Regeneron Science Talent Search Competition & 2018 \\
& \\
Ad-hoc journal referee for Nature, Molecular Cell, Reviewer Commons, PNAS, PLoS Biology, eLife, Cell Reports, Nature Communications, Nature Microbiology, Physical Biology, Physical Review Letters, Physical Review B & \\
& \\
Ad-hoc grant reviewer for US National Science Foundation, US-Israel Binational Foundation, Research Foundation --- Flanders Belgium, Swedish Foundation for Strategic Research, Israel Ministry of Science and Technology, Swiss National Science Foundation & \\
\end{tabular}

# Extramural Research Support

\begin{tabular}{>{\raggedright\arraybackslash}p{6.6in} >{\raggedleft\arraybackslash}p{0.6in}} 
\uline{Current} & \\
& \\
NIH R35 GM119835 (PI) \newline 
Regulation of Protein Synthesis by Synonymous Codon Usage \newline
Current Direct Costs: \$241,973/yr
& 2016--26 \\
& \\
NIH R01 AT012826 (Co-PI) \newline
Unraveling Microprotein Biology with an Evolutionary-Immunological Framework \newline
Current Direct Costs to Subramaniam lab: \$225,000/yr 
& 2023--28 \\
& \\
\uline{Completed} & \\
& \\
NSF CAREER MCB-1846521 (PI) \newline Experimentally Integrated Modeling of Quality Control During Eukaryotic mRNA Translation & 2019--24 \\
& \\
Sidney Kimmel Scholarship (PI) \newline Quantitative Profiling of Synonymous Mutation Effects in Cancer Cells & 2017--19 \\
& \\
NIH K99/R00 GM107113 (PI)\newline Role of Synonymous Codon Usage as Gene Regulators in Bacteria and Cancer Cells & 2013--17 \\
\end{tabular}


# Preprints & Publications

**^†^** indicates *corresponding author; Subramaniam lab members are underlined.*

::: {.reverse}
1. [Bakker RA]{.underline}, Nicholson OB, [Park H]{.underline}, Xiao YL, Tang W, \textcolor[RGB]{128,0,128}{Subramaniam AR} **^†^**, Lapointe CP **^†^**.
   *Deaminase-based RNA recording enables high throughput mutational profiling of protein-RNA interactions.* \textcolor[RGB]{0,100,0}{\textbf{bioRxiv}} 2025.04.11.648485, \textcolor[RGB]{0,100,0}{\textbf{(2025)}}. [PMCID: 12027372](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12027372/)

2. [Nugent PJ]{.underline}, [Park H]{.underline}, Wladyka CL, [Yelland J]{.underline}, Sinha S, [Chen KY]{.underline}, Quarterman G, Bynum C, Lee SC, Hsieh AC, \textcolor[RGB]{128,0,128}{Subramaniam AR} **^†^**.
   *Decoding post-transcriptional regulatory networks by RNA-linked CRISPR screening in human cells.* \textcolor[RGB]{0,100,0}{\textbf{Nature Methods}} 22:1237 \textcolor[RGB]{0,100,0}{\textbf{(2025)}}. [PMID: 40442371](https://pubmed.ncbi.nlm.nih.gov/40442371/)

3. [Chen KY]{.underline}, [Toro-Moreno M]{.underline} **^†^**, \textcolor[RGB]{128,0,128}{Subramaniam AR} **^†^**.
   *GitHub enables collaborative and reproducible laboratory research.* \textcolor[RGB]{0,100,0}{\textbf{PLoS Biology}} 23(2)\:e3003029 \textcolor[RGB]{0,100,0}{\textbf{(2025)}}. [PMCID: 11828340](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11828340/)

4. [Chen KY]{.underline}, [Park H]{.underline}, \textcolor[RGB]{128,0,128}{Subramaniam AR} **^†^**.
   *Massively parallel identification of sequence motifs triggering ribosome-associated mRNA quality control.* \textcolor[RGB]{0,100,0}{\textbf{Nucleic Acids Research}} 52:7171 \textcolor[RGB]{0,100,0}{\textbf{(2024)}}. [PMCID: 11229359](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11229359/)

5. Hou W, Harjono V, Harvey AT, \textcolor[RGB]{128,0,128}{Subramaniam AR}, Zid BM **^†^**.
   *Quantification of elongation stalls and impact on gene expression in yeast.* \textcolor[RGB]{0,100,0}{\textbf{RNA}} 29:1928 \textcolor[RGB]{0,100,0}{\textbf{(2023)}}. [PMCID: 10653389](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10653389/)

6. Jana S, Brahma S, Arora S, Wladyka CL, Hoang P, Blinka S, Hough R, Horn JL, Liu Y, Wang LJ, Depeille P, Smith E, Montgomery RB, Lee JK, Haffner MC, Vakar-Lopez F, Grivas P, Wright JL, Lam HM, Black PC, Roose JP, Ryazanov AG, \textcolor[RGB]{128,0,128}{Subramaniam AR}, Henikoff S, Hsieh AC **^†^**.
   *Transcriptional-translational conflict is a barrier to cellular transformation and cancer progression.* \textcolor[RGB]{0,100,0}{\textbf{Cancer Cell}} 41:853 \textcolor[RGB]{0,100,0}{\textbf{(2023)}}. [PMCID: 10208629](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10208629/)

7. [Burke PC]{.underline}, [Park H]{.underline}, \textcolor[RGB]{128,0,128}{Subramaniam AR} **^†^**.
   *A nascent peptide code for translational control of mRNA stability in human cells.* \textcolor[RGB]{0,100,0}{\textbf{Nature Communications}} 13:6829 \textcolor[RGB]{0,100,0}{\textbf{(2022)}}. [PMCID: 9652226](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9652226/)

8. [Bottorff TA]{.underline}, [Park H]{.underline}, Geballe AP **^†^**, \textcolor[RGB]{128,0,128}{Subramaniam AR} **^†^**.
   *Translational buffering by ribosome stalling in upstream open reading frames.* \textcolor[RGB]{0,100,0}{\textbf{PLoS Genetics}} 18(10)\:e1010460 \textcolor[RGB]{0,100,0}{\textbf{(2022)}}. [PMCID: 9648851](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9648851/)

9. Farooq Z, Kusuma F, [Burke P]{.underline}, Dufour CR, Lee D, Tabatabaei N, Toboz P, Radovani E, Greenblatt JF, Rehman J, Class J, Khoutorsky A, Fonseca BD, Richner JM, Mercier E, Bourque G, Giguère V, \textcolor[RGB]{128,0,128}{Subramaniam AR}, Han J, Tahmasebi S **^†^**.
   *The amino acid sensor GCN2 suppresses terminal oligopyrimidine (TOP) mRNA translation via La-related protein 1 (LARP1).* \textcolor[RGB]{0,100,0}{\textbf{Journal of Biological Chemistry}} 298:102277 \textcolor[RGB]{0,100,0}{\textbf{(2022)}}. [PMCID: 9396407](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9396407/)

10. Clough CA, Pangallo J, Sarchi M, Ilagan JO, North K, Bergantinos R, Stolla MC, Naru J, [Nugent P]{.underline}, Kim E, Stirewalt DL, \textcolor[RGB]{128,0,128}{Subramaniam AR}, Abdel-Wahab O, Abkowitz JL, Bradley RK **^†^**, Doulatov S **^†^**.
    *Coordinated missplicing of TMEM14C and ABCB7 causes ring sideroblast formation in SF3B1-mutant myelodysplastic syndrome.* \textcolor[RGB]{0,100,0}{\textbf{Blood}} 139:2038 \textcolor[RGB]{0,100,0}{\textbf{(2022)}}. [PMCID: 8972092](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8972092/)

11. [Kriner MA]{.underline}, \textcolor[RGB]{128,0,128}{Subramaniam AR} **^†^**.
    *The serine transporter SdaC prevents cell lysis upon glucose depletion in Escherichia coli.* \textcolor[RGB]{0,100,0}{\textbf{MicrobiologyOpen}} 9\:e960 \textcolor[RGB]{0,100,0}{\textbf{(2020)}}. [PMCID: 7002108](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7002108/)

12. Green R, Sonal, Wang L, Hart SFM, Lu W, Skelding D, Burton JC, Mi H, Capel A, Chen HA, Lin A, \textcolor[RGB]{128,0,128}{Subramaniam AR}, Rabinowitz JD, Shou W **^†^**.
    *Metabolic excretion associated with nutrient-growth dysregulation promotes the rapid evolution of an overt metabolic defect.* \textcolor[RGB]{0,100,0}{\textbf{PLoS Biology}} 18\:e3000757 \textcolor[RGB]{0,100,0}{\textbf{(2020)}}. [PMCID: 7470746](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7470746/)

13. [Machkovech HM]{.underline}, Bloom JD **^†^**, \textcolor[RGB]{128,0,128}{Subramaniam AR} **^†^**.
    *Comprehensive profiling of translation initiation in influenza virus infected cells.* \textcolor[RGB]{0,100,0}{\textbf{PLoS Pathogens}} 15(1)\:e1007518 \textcolor[RGB]{0,100,0}{\textbf{(2019)}}. [PMCID: 6361465](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6361465/)

14. [Park H]{.underline}, \textcolor[RGB]{128,0,128}{Subramaniam AR} **^†^**.
    *Inverted translational control of eukaryotic gene expression by ribosome collisions.* \textcolor[RGB]{0,100,0}{\textbf{PLoS Biology}} 17(9)\:e3000396 \textcolor[RGB]{0,100,0}{\textbf{(2019)}}. [PMCID: 6750593](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6750593/)

15. Darnell AM, \textcolor[RGB]{128,0,128}{Subramaniam AR} **^†^**, O’Shea EK **^†^**.
    *Translational control through differential ribosome pausing during amino acid limitation in mammalian cells.* \textcolor[RGB]{0,100,0}{\textbf{Molecular Cell}} 71:229 \textcolor[RGB]{0,100,0}{\textbf{(2018)}}. [PMCID: 6516488](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6516488/)

16. [Ferrin MA]{.underline}, \textcolor[RGB]{128,0,128}{Subramaniam AR} **^†^**.
    *Kinetic modeling predicts a stimulatory role for ribosome collisions at elongation stall sites in bacteria.* \textcolor[RGB]{0,100,0}{\textbf{eLife}} 6\:e23629 \textcolor[RGB]{0,100,0}{\textbf{(2017)}}. [PMCID: 5446239](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5446239/)
:::

\pagebreak
# Prior to 2015

:::{.reverse}
1. [Subramaniam AR]{.underline}, Zid B, [O'Shea EK]{.underline}. An integrated approach reveals regulatory controls on bacterial translation elongation. *Cell* 159:1200 (2014). [PMCID: 4243059](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4243059/)

1. Subramaniam AR, DeLoughery A, Bradshaw N, Chen Y, O’Shea EK, [Losick R]{.underline}, [Chai Y]{.underline}. A serine sensor for multicellularity in a bacterium. *eLife* 2:e01501 (2013). [PMCID: 3862929](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3862929/) 

1. Subramaniam AR, Pan T, [Cluzel P]{.underline}. Environmental perturbations lift the degeneracy of the genetic code to regulate protein levels in bacteria. *PNAS* 110:2419 (2013). [PMCID: 3568297](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3568297/)

1. Obuse H, Subramaniam AR, Furusaki A, Gruzberg IA, Ludwig AWW. Conformal invariance, multifractality, and finite-size scaling at Anderson localization transitions in two dimensions. *Physical Review B* 82:035309 (2010). [DOI](https://doi.org/10.1103/PhysRevB.82.035309)
 
1.  Subramaniam AR, Gruzberg IA, Ludwig AWW. Boundary criticality and multifractality at the 2D spin quantum Hall transition. *Physical Review B* 78:245105 (2008). [DOI](https://doi.org/10.1103/PhysRevB.78.245105)

1.  Obuse H, Subramaniam AR, Furusaki A, Gruzberg IA, Ludwig AWW. Boundary multifractality at the integer quantum Hall plateau transition: implications for the critical theory. *Physical Review Letters* 101:116802 (2008). [DOI](https://doi.org/10.1103/PhysRevLett.101.116802)

1.  Obuse H, Subramaniam AR, Furusaki A, Gruzberg IA, Ludwig AWW. Corner multifractality for reflex angles and conformal invariance at 2D Anderson metal-insulator transition with spin-orbit scattering. *Physica E* 40:1404 (2008). [DOI](https://doi.org/10.1016/j.physe.2007.09.024)

1.  Jia X, Subramaniam AR, Gruzberg IA, Chakravarty S. Entanglement entropy and multifractality at localization transitions. *Physical Review B* 77:014208 (2008). [DOI](https://doi.org/10.1103/PhysRevB.77.014208)

1.  Obuse H, Subramaniam AR, Furusaki A, Gruzberg IA, Ludwig AWW. Multifractality and conformal invariance at 2D metal-insulator transition in the spin-orbit symmetry class. *Physical Review Letters* 98:156802 (2007). [DOI](https://doi.org/10.1103/PhysRevLett.98.156802)

1.  Mildenberger A, Subramaniam AR, Narayanan R, Evers F, Gruzberg IA, Mirlin AD. Boundary multifractality in critical 1D systems with long-range hopping. *Physical Review B* 75:094204 (2007). [DOI](https://doi.org/10.1103/PhysRevB.75.094204)

1.  Subramaniam AR, Gruzberg IA, Ludwig AWW, Evers F, Mildenberger A, Mirlin AD. Surface criticality and multifractality at localization transitions. *Physical Review Letters* 96:126802 (2006). [DOI](https://doi.org/10.1103/PhysRevLett.96.126802)
:::
